Bitbucket.Authentication/Src/Authentication.cs

```diff
@@ -53,15 +54,16 @@ public class Authentication : BaseAuthentication, IAuthentication
             RuntimeContext context,
             ICredentialStore personalAccessTokenStore,
             AcquireCredentialsDelegate acquireCredentialsCallback,
-            AcquireAuthenticationOAuthDelegate acquireAuthenticationOAuthCallback)
+            AcquireAuthenticationOAuthDelegate acquireAuthenticationOAuthCallback,
+            IAuthority authority = null)
```

Comment:
Add the ability to override the default Authority. Allows for mocking and testing elsewhere.